src/app.css

```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700');
```

Comment:
It applies the same for CSS also 
can create another component and write a child component